src/main/webapp/app/shared/components/confirm-autofocus-button.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { NgbActiveModal, NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { ModalConfirmAutofocusComponent } from 'app/intellij/modal-confirm-autofocus/modal-confirm-autofocus.component';
+
+@Component({
+    template: `
+        <div class="modal-header">
+            <h4 class="modal-title">{{ title | translate }}</h4>
+            <button type="button" class="close" aria-label="Close button" aria-describedby="modal-title" (click)="modal.dismiss('Cross click')">
+                <span aria-hidden="true">&times;</span>
+            </button>
+        </div>
+        <div class="modal-body">
+            <p>{{ text | translate }}</p>
+        </div>
+        <div class="modal-footer">
+            <button type="button" class="btn btn-outline-secondary" (click)="modal.dismiss('cancel click')">Cancel</button>
```

Comment:
do we need translations here?

Reply:
I would argue that "Cancel" and "Ok" are both valid in German and English. Although, if there will be other languages in the future, those values won't show up. I'll add translations so that we are safe